core/src/main/java/io/questdb/std/Chars.java

```diff
@@ -468,26 +468,45 @@ public static String stringFromUtf8Bytes(long lo, long hi) {
         return b.toString();
     }
 
-    public static String toLowerCaseAscii(@Nullable CharSequence value) {
-        if (value == null) {
-            return null;
+    public static void toLowerCaseAscii(@Nullable final CharSequence value, final CharSink sink) {
+        if (value == null || value.length() == 0) {
+            return;
         }
-        final int len = value.length();
-        if (len == 0) {
-            return "";
+
+        for (int i = 0; i < value.length(); i++) {
+            sink.put(toLowerCaseAscii(value.charAt(i)));
         }
+    }
 
-        final CharSink b = Misc.getThreadLocalBuilder();
-        for (int i = 0; i < len; i++) {
-            b.put(toLowerCaseAscii(value.charAt(i)));
+    public static String toLowerCaseAscii(@Nullable final CharSequence value) {
+        CharSink sink = Misc.getThreadLocalBuilder();
+        toLowerCaseAscii(value, sink);
+        return sink.toString();
+    }
+
+    public static char toLowerCaseAscii(char character) {
```

Comment:
`toLowerCaseAscii()` is used elsewhere, where only ascii characters allowed. We need to keep implementation unchanged.  The SQL to_lowercase() should call `Character.toLowerCase()` directly.

Calling Character.toLowerCase() from toLowerCaseAscii() is also misleading, because implementation is no longer "ascii" specific.

Reply:
As `Character.toLowerCase()` is a superset of what the `toLowercaseAscii(char)` was doing I thought it was ok just to call it inside `toLowercaseAscii(char)`.

I'll create new methods `toUppercase(...)` and `toLowercase(...)` to use `Character.toUpperCase()` and `Character.toLowerCase()` respectively.